docs/include/conf/portland/cfp.rst

```diff
@@ -0,0 +1,115 @@
+accepted by 1st March.
+
+Conference goals
+----------------
+
+The purpose of Write the Docs is to bring together people who are passionate
+about documentation and give them an opportunity to interact with and learn from
+their peers.
+
+One of the defining characteristics of our community is the broad range of
+professional backgrounds we hail from—and our conference seeks to celebrate
+that. Whether your job title is technical writer, programmer, product manager,
+developer evangelist, support advocate, or something else entirely, we want to
+hear from you. Bring us your tips, tricks, experiences, ideas, frustrations,
+hopes, or anything else that your passion for documentation inspires you to
+share!
+
+What we’re looking for
+----------------------
+
+Although the focus of Write the Docs is on documentation, particularly within
+the software industry, we aim for a wide range of talks that will lead people
+in new directions. We’re after talks that deliver a fresh perspective on or
+demonstrate a creative approach to writing the docs.
+
+We encourage speakers at all levels to submit talks. First-time speakers are
+great. Experienced speakers are great. Hearing the perspective of a novice
+documentarian can be just as valuable as a deep-dive from an industry veteran.
+
+We love talks that deliver hard-earned knowledge based on real-world experience
+and research. We also get really excited about talks that make people look at
+their craft in a different light. We are always delighted with the breadth of
+knowledge that our community brings together.
+
+We tend to avoid talks about specific tools, no matter how awesome they are.
+We much prefer talks about process and culture shifts,
+and the tools can play a supporting role.
+
+If you already have an idea for what you'd like to speak about, awesome! Pitch it to us!
+
+Need some more examples?
+Check out our previous abstracts for some ideas:
+
+* `Portland 2016 <http://www.writethedocs.org/conf/na/2016/speakers/>`_
+* `Portland 2015 <http://www.writethedocs.org/conf/na/2015/speakers/>`_
+* `Prague 2016 <http://www.writethedocs.org/conf/eu/2016/speakers/>`_
+* `Prague 2015 <http://www.writethedocs.org/conf/eu/2015/speakers/>`_
+
+
+Presentation format
+-------------------
+
+Presentations will be scheduled in 30 minute blocks. We won't be doing Q&A after
+the talks, but we ask that speakers be available at the front of the stage
+for a few minutes of the break after their talk to answer questions.
+
+**NEW THIS YEAR**: We're also looking for a few speakers who'd be interested in facilitating a group discussion,
+rather than giving a formal talk, as part of this year's Unconference.
+For the past few years, we've held unconf sessions downstairs from the main hall. To increase folks' awareness of and interest in the unconf,
+we're looking to get a handful of pre-planned anchor sessions (also 30 minutes) to get the ball rolling.
```

Comment:
whoops! how did this get in here? 🤔 Seems to be the 2017 CFP file, yeah? But we do seem to have the 2018 one up as well, so... I'm confused. 😬